Description: Add XML includes for local database config
Author: Markus Frosch <lazyfrosch@debian.org>
Last-Update: 2014-09-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/etc/conf.d/databases.xml.in
+++ b/etc/conf.d/databases.xml.in
@@ -13,6 +13,11 @@
 <databases xmlns:db="http://agavi.org/agavi/config/parts/databases/1.0" xmlns:ae="http://agavi.org/agavi/config/global/envelope/1.0">
 
 
+    <!-- comment this if you do not want to use debconf with dbconfig-common -->
+    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="/etc/icinga-web/conf.d/database-web.xml#xpointer(databases/node())">
+        <xi:fallback></xi:fallback>
+    </xi:include>
+
     <!-- 
     Overwrite the icinga-web database (where users, roles, views, etc are stored 
     -->
@@ -41,6 +46,11 @@
     </db:database>
     -->
 
+    <!-- comment this if you do not want to use debconf with dbconfig-common for the IDO database -->
+    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="/etc/icinga-web/conf.d/database-ido.xml#xpointer(databases/node())">
+        <xi:fallback></xi:fallback>
+    </xi:include>
+
     <!-- 
         Overwrite the default database conneciton for icinga data retrieval
         Make sure to use IcingaDoctrineDatabase, otherwise the use_retained parameter will be ignored
